data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/tracing/TracingSpan.java

```diff
@@ -26,6 +27,8 @@ public class TracingSpan {
     private static final AttributeKey<String> MESSAGING_MESSAGE_ID = stringKey("messaging.message_id");
     private static final AttributeKey<String> MESSAGING_MESSAGE_SOURCE = stringKey("messaging.message_source");
     private static final AttributeKey<String> MESSAGING_MESSAGE_TYPE = stringKey("messaging.message_type");
+    private static final AttributeKey<String> EGRESS_UUID = stringKey("egress.uuid");
+    private static final AttributeKey<String> EGRESS_NAME = stringKey("egress.name");
```

Comment:
Can we use `consumer_name` and `consumer_uuid` so that it's not using internal terminology that end users are not familiar with? 

It will also help with being consistent with metrics tags

Reply:
sure, that's a much better name.

Reply:
I renamed egress to consumer. thanks